openmls/src/group/tests_and_kats/tests/proposal_validation.rs

```diff
@@ -1042,10 +1038,10 @@ fn test_valsem105() {
     for key_package_version in [
         KeyPackageTestVersion::WrongCiphersuite,
         KeyPackageTestVersion::WrongVersion,
-        KeyPackageTestVersion::UnsupportedVersion,
-        KeyPackageTestVersion::UnsupportedCiphersuite,
         KeyPackageTestVersion::ValidTestCase,
     ] {
+        println!("running test {key_package_version:?}");
```

Comment:
Drop debug print.

Reply:
Are you sure? This is in a test (so it doesn't print to stdout in operation) and we are basically running sub-tests, and it's often difficult to understand which sub-test is failing. IMHO when running sub-tests it makes sense to print which one we are running now.